.env.example

```
@@ -0,0 +1,95 @@
+# CoinGecko API key for fetching GNO/USD price (used to derive CRC/USD)
+# Get a free key at: https://www.coingecko.com/api/pricing
+# Free tier: 30 requests/min, 10,000 requests/month
+COINGECKO_API_KEY=your-coingecko-api-key-here
+
+# Fallback CRC price in USD (used if CoinGecko API is unavailable)
+# This is a manual estimate - update based on GNO offer price
+PRICING_FALLBACK_CRC_PRICE_USD=0.01
+
+# Price cache expiry in minutes (default: 5)
+# Increase to reduce API calls, decrease for more real-time prices
+PRICING_CACHE_EXPIRY_MINUTES=5
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)